test/functions.php

```diff
@@ -0,0 +1,23 @@
+<?php
+require('../functions.php');
+use PHPUnit\Framework\TestCase;
+class collection_appTest extends TestCase
+{
+     public function testSuccessMakePlantEntryTile () {
```

Comment:
Ok so I think that you're trying to make sure that output is correct i dont think you can as its dependant on the database. i don't think this is really what unit testing is means for. It's more for making sure that it is used with proper data types and such. (check with mike)